tool/testsrv.c

```diff
@@ -42,12 +44,19 @@ void sigcont_handler (int signum) {
 }
 
 int main (int argc, char **argv) {
-	int sox, cnx;
+	int sox, cnx, rc;
 	int plainfd;
-	struct sockaddr_in6 sin6;
+	const char *host = argc < 2 ? "::" : argv[1];
+	const char *service = argc < 3 ? "12345" : argv[2];
+	struct addrinfo	*res;
 	sigset_t sigcontset;
 	uint8_t rndbuf [16];
 
+	rc = getaddrinfo(host, service, NULL, &res);
```

Comment:
dit is niet meer nodig; ik heb een andere socket.c module toegevoegd die ongeveer dit doet, maar nog wat netter; heb ik gejat uit KXOVER waar ik wat zuiverder was.  is ook IPv4 en IPv6 capable, daar deed je dit voor meen ik.

enfin, deze hele file komt volgens mij voort uit een merge conflict gedoe.  deze hele tweede commit kan weg.  de eerste lijkt me geen enkel probleem als die bij jou werkt.